.scalafmt.conf

```diff
@@ -13,8 +13,11 @@ To fix this problem:
 project.excludeFilters = [
   "test-workspace"
   "metals-bench/src/main/resources"
-  "mtags/src/main/scala-0.26/"
-  "mtags/src/main/scala-0.27/"
+  "mtags/src/main/scala-3.0/"
+  "mtags/src/main/scala-3.0.0-M1/"
```

Comment:
Ignored this for now, so that we can format those directories manually using scala3 dialect. There is still a bunch of work on scalafmt to make it all work out of the box.